x-pack/performance/journeys/dashboard_listing_page.ts

```diff
@@ -29,4 +29,13 @@ export const journey = new Journey({
     await page.click('[data-test-subj="deleteSelectedItems"]');
     await page.click('[data-test-subj="confirmModalConfirmButton"]');
     await page.waitForSelector(`[data-test-subj="table-is-ready"]`);
+  })
+  .step('Add  dashboard', async ({ page, inputDelays }) => {
+    await page.click('[data-test-subj="newItemButton"]');
+    await page.click('[data-test-subj="dashboardSaveMenuItem"]');
+    await page.type('[data-test-subj="savedObjectTitle"]', 'foobar dashboard', {
+      delay: inputDelays.TYPING,
+    });
+    await page.click('[data-test-subj="confirmSaveSavedObjectButton"]');
+    await page.locator('text="Editing foobar dashboard"');
```

Comment:
@dmlemeshko I believe this would be a proper way to wait until save has completed and changes are applied on the dashboard, but would be great to have confirmation.

Reply:
Hey @thomasneirynck 

It looks we don't have `saved_object_create_time` event in code or I miss the purpose of this new step?

<img width="1583" alt="image" src="https://user-images.githubusercontent.com/10977896/212375825-2cc3113e-a08a-4405-963f-78fd59bbe08d.png">

As for waiting, I think we can use the same approach we do in our functional tests relying on appearing toast message? 
```
await page.locator('[data-test-subj="saveDashboardSuccess"]');
```

Do you know if it appears after SO was created or still while operation is in progress?

Reply:
Thanks, perfect, made the change. 

> Do you know if it appears after SO was created or still while operation is in progress?

I believe it appears after the saving has completed/